docs/docs/resources/pipeline-defaults/defaults.yaml

```diff
@@ -0,0 +1,190 @@
+    repoAuthFlags: 
+      username: user
+      password: pass
+      ca_file: /home/user/path/to/ca-file
+      insecure_skip_tls_verify: false
+  version: 1.0.0 # Helm chart version
+#
+# Base component for Kafka-based components.
+#
+# Parent of: ProducerApp, StreamsApp
+# Child of: KubernetesApp
+kafka-app:
+  app:
+    streams:
+      brokers: ${broker}
+      schemaRegistryUrl: ${schema_registry_url}
+    nameOverride: override-with-this-name
+#
+# StreamsApp component that configures a streams bootstrap app.
+#
+# Child of: KafkaApp
+# More documentation on StreamsApp: https://github.com/bakdata/streams-bootstrap
+streams-app:
+  # No arbitrary keys are allowed under `app`.
+  # Allowed configs:
+  # https://github.com/bakdata/streams-bootstrap/tree/master/charts/streams-app
+  app:
+    # Streams Bootstrap streams section
+    streams:
+      inputTopics:
+        - topic1
+        - topic2
+      outputTopic: output-topic
+      inputPattern: input-pattern
+      extraInputTopics:
+        input_role1:
+          - input_topic1
+          - input_topic2
+        input_role2:
+          - input_topic3
+          - input_topic4
+      extraInputPatterns:
+        pattern_role1: input_pattern1
+      extraOutputTopics: 
+        output_role1: output_topic1
+        output_role2: output_topic2
+      errorTopic: error-topic
+      config: 
+        my.streams.config: my.value
+    autoscaling:
+      consumerGroup: consumer-group
+      lagThreshold: 0 # Average target value to trigger scaling actions.
+      enabled: false # Whether to enable auto-scaling using KEDA.
+      # This is the interval to check each trigger on.
+      # https://keda.sh/docs/2.7/concepts/scaling-deployments/#pollinginterval
+      pollingInterval: 30
+      # The period to wait after the last trigger reported active before scaling
+      # the resource back to 0. https://keda.sh/docs/2.7/concepts/scaling-deployments/#cooldownperiod
+      cooldownPeriod: 300
+      # The offset reset policy for the consumer if the the consumer group is
+      # not yet subscribed to a partition.
+      offsetResetPolicy: earliest
+      # This setting is passed to the HPA definition that KEDA will create for a
+      # given resource and holds the maximum number of replicas of the target resouce.
+      # https://keda.sh/docs/2.7/concepts/scaling-deployments/#maxreplicacount
+      maxReplicas: 1
+      # If this property is set, KEDA will scale the resource down to this
+      # number of replicas.
+      # https://keda.sh/docs/2.7/concepts/scaling-deployments/#idlereplicacount
+      idleReplicas: 0
+      topics: # List of auto-generated Kafka Streams topics used by the streams app.
+        - topic1
+        - topic2
+#
+# Holds configuration to use as values for the streams bootstrap producer Helm
+# chart.
+#
+# Child of: KafkaApp
+# More documentation on ProducerApp: https://github.com/bakdata/streams-bootstrap
+producer:
+  # No arbitrary keys are allowed under `app`.
```

Comment:
I think extra keys are allowed https://github.com/bakdata/kpops/blob/a5c7543ef59f27899342c0b3eb157036eaeb78da/kpops/components/streams_bootstrap/producer/model.py#L18

Reply:
I meant that as "no straying away from the streams-bootstrap configs". Is that correct and if yes, how should I word it better?

Reply:
I think your description is sufficient here
> Allowed configs:
>  https://github.com/bakdata/streams-bootstrap/tree/master/charts/producer-app

Reply:
So I shouldn't change anything?

Reply:
I would suggest to remove this line "# No arbitrary keys are allowed under `app`."
and only keep the rest of the description

Reply:
done